vite.config.js

```diff
@@ -1,7 +1,9 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
+import svgr from "vite-plugin-svgr"
```

Comment:
We decided to remove it, didnt we?

Reply:
Yes, of course, I just didn't know that this line should be removed)) I no longer use this library in components. I import directly SVG. Thanks for checking and advice :)